app/helpers/application_helper.rb

```diff
@@ -26,4 +26,12 @@ def state_label(s)
   def minister_warning?(question, minister)
     !!(question.present? && question.open? && minister.try(:deleted?))
   end
+
+  def show_inactive_link(show_inactive)
+    if show_inactive   
+      link_to ('View active action officers'), action_officers_path, {class: 'button-secondary'}
```

Comment:
The parentheses aren't needed here or below
```suggestion
      link_to 'View active action officers', action_officers_path, { class: 'button-secondary' }
```

Reply:
Removed brackets and added space in {} brackets